tests/test_core.py

```diff
@@ -1757,7 +1757,7 @@ def test_memorygrowth(self):
     self.do_run(src, '*pre: hello,4.955*\n*hello,4.955*\n*hello,4.955*')
     win = open('src.cpp.o.js').read()
 
-    if '-O2' in self.emcc_args:
+    if '-O2' in self.emcc_args and not Settings.WASM:
```

Comment:
can use `self.is_wasm()` for these

Reply:
Sounds reasonable.  I'm not sure what the difference is.  Perhaps we should change the occurrences of Settings.WASM in this file too (in a followup PR).